models/pull.go

```diff
@@ -270,6 +270,8 @@ const (
 	MergeStyleMerge MergeStyle = "merge"
 	// MergeStyleRebase rebase before merging
 	MergeStyleRebase MergeStyle = "rebase"
+	// MergeStyleRebaseMergeCommit rebase before merging with merge commit (--no-ff)
+	MergeStyleRebaseMergeCommit MergeStyle = "rebase-merge-commit"
```

Comment:
`MergeStyleRebaseMerge` would be better name

Reply:
Agree, will change it.

Reply:
This works for all places, but the locale file:
```
pulls.rebase_merge_pull_request = Rebase and Merge
pulls.rebase_merge_commit_pull_request = Rebase and Merge (--no-ff)
```
The `MergeStyleRebase` already uses the (potential) name of the new `MergeStyleRebaseMerge`.